src/test/resources/application.conf

```diff
@@ -23,6 +23,14 @@ constellation {
   dataPollingManagerOn = false
   storage {
     enabled = false
+    aws {
+      enabled = true
+      bucket-name = "bucketA"
+      region = "region"
+      aws-access-key = "awsAccessKey"
+      aws-secret-key = "awsSecret"
+      backup-buckets-names = [bucketB, bucketC]
```

Comment:
I would rather just keep `bucket-names` and rely on order. But up to you. Because in general we do not have concept of "main" and "secondary" buckets. All buckets are equal and we could even do race or random queries there.

Reply:
I wanted to not change the current config structure - if someone else is relying on it other than us (probably not). But also I wanted to enforce at least one bucket being specified when AWS storage is enabled and not handle empty list case. If backup-buckets are not specified I'm simply defaulting to Nil for backups.